packages/cra-template-apps-typescript-starter-kit/template.json

```diff
@@ -0,0 +1,33 @@
+{
+  "package": {
+    "dependencies": {
+      "@datadog/ui-extensions-react": "0.30.1",
+      "@datadog/ui-extensions-sdk": "0.30.1",
+      "milligram": "^1.4.1",
+      "react": "^17.0.2",
+      "react-dom": "^17.0.2",
```

Comment:
Relevant details in [this comment](https://github.com/DataDog/apps/pull/127/files#r839151925).